lib/perl/Genome/FeatureList/Command/DumpIntervalList.pm

```diff
@@ -38,6 +43,9 @@ class Genome::FeatureList::Command::DumpIntervalList {
 sub execute {
     my $self = shift;
 
+#    if ($self->preserve_region_names and $self->merge) {
```

Comment:
If this is going to be commented out, it should probably just be deleted?

Reply:
Oops. It totally should!